lib/src/lib/table/components/table-cell-value/table-cell-value.component.ts

```diff
@@ -56,4 +56,13 @@ export class Lab900TableCellValueComponent<T = any> implements OnChanges {
   public getMatTooltip(): string {
     return this.cell.cellTooltip ? this.cell.cellTooltip(this.data, this.cell) : '';
   }
+
+  public getMatTooltipOption(): MatTooltipOptions {
+    const matTooltipOption = this.cell.cellTooltipOptions ? this.cell.cellTooltipOptions : {};
+
+    return {
+      ...matTooltipOption,
+      tooltipPosition: matTooltipOption.tooltipPosition ?? 'below',
```

Comment:
@lab900-frederico-teixeira Can you also get this from the TooltipPosition enum?

Reply:
I think, the typings are the only thing available, at least I didn't found then.

Reply:
Right, it's not really an enum, just typed as those strings. Fine with me then.